src/default_config_params/mod.rs

```diff
@@ -0,0 +1,66 @@
+use std::path::PathBuf;
+pub fn get_default_wallet_path() -> PathBuf {
```

Comment:
Instead of `default_wallet_path` you need a function called `get_default_datadir`. Which will always return `~/.teleport`. 

You need a little logic to identify the home folder of the system and append `.teleport` to it. Thats it.

You can do that using the `dirs` library, its already in our cargo, but i bet theres a non dependent way of getting the home dir. We wanna throw away the `dir` lib later on.

Reply:
the `dirs` crate has the home_dir call that returns a Some<PathBuf> for different OSes. So there really isn't much logic that would go into confirming this. My other attempt of using ENV variables would be limited to UNIX systems. Any reason we'd want to move away from `dirs`? It would assist in dealing with the complexity of  different config file locations in the long run.